### DISCUSSION TITLE
Open color picker from a button

### DISCUSSION THREAD

**Discussion by TBTech205** [mdbassit/Coloris#112] [General]
So I would like to open the color picker from a button not a input field
Then I can add the color preview some where else one the page using the onChange event

**Comment by mdbassit** [comment]
This color picker can only be opened from an input field, but you can use CSS to style it to look like a button if. you like.

If you don't want the color preview button to be added to your field, set the `wrap` option to false when initializing Coloris.

**Reply by TBTech205** [reply]
Hey, 

I was able to get the none button to work.. Could this be a thing for the future?

**Reply by mdbassit** [reply]
That is unlikely. For my own personal use, I need the color picker to be attached to input fields. But I'm glad you were able to solve your problem.
